src/components/EtherscanLink.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0'
+
+import { EtherscanLink, EtherscanLinkProps } from 'components/EtherscanLink'
+import { Network } from 'types'
+
+const networkIds = Object.values(Network).filter(Number.isInteger)
+
+export default {
+  title: 'Components/EtherscanLink',
+  component: EtherscanLink,
+  argTypes: {
+    label: { control: 'text' },
+    networkId: { control: { type: 'inline-radio', options: networkIds } },
+  },
```

Comment:
What if you don't have this?
Storybook should be smart enough to get the arg types from component props.
At least most of the time.

Reply:
I understand u don't get this nice network id suggestions :)

Reply:
![image](https://user-images.githubusercontent.com/2352112/91589187-a05c8300-e959-11ea-8226-36e0975de86b.png)
Maybe label can be removed! I will do that

Reply:
Amending my last comment, if you don't say the "label" is a test, we need to quote it to test it. I guess is because of this:

![image](https://user-images.githubusercontent.com/2352112/91589625-56c06800-e95a-11ea-8dea-7c91d9d45368.png)

Reply:
Understood